crud/src/main/java/com/redhat/lightblue/rest/crud/cmd/FindCommand.java

```diff
@@ -152,15 +157,17 @@ public CallStatus run() {
             // Until streaming is supported in mediator, we'll get the
             // results and stream them
             if(stream) {
-                streamResponse=getMediator().findAndStream(ireq);
+                streamResponse=getMediator().findAndStream(ireq, metricCtx);
                 return new CallStatus(new Response());
             } else {
-                return new CallStatus(getMediator().find(ireq));
+                return new CallStatus(getMediator().find(ireq, metricCtx));
             }
         } catch (Error e) {
```

Comment:
I think we also need to mark errors for conditions above:

```
             try {
                 ireq = getJsonTranslator().parse(FindRequest.class, JsonUtils.json(request));
             } catch (Exception e) {
                 LOGGER.error("find:parse failure: {}", e);
                 return new CallStatus(Error.get(RestCrudConstants.ERR_REST_FIND, "Error during the parse of the request"));
             }
             LOGGER.debug("Find request:{}", ireq);
             try {
                 validateReq(ireq, entity, version);
             } catch (Exception e) {
                 LOGGER.error("find:validate failure: {}", e);
                 return new CallStatus(Error.get(RestCrudConstants.ERR_REST_FIND, "Request is not valid"));
             }
```

Similar errors can probably occur in other commands, too.